.circleci/config.yml

```diff
@@ -113,33 +113,6 @@ jobs:
             codecov --file ../karma_coverage_reports/lcov.info -F frontend
           when: on_success
 
-  backend_tests:
```

Comment:
Don't delete this yet:

1. Introduce the GitHub Actions flow in this PR, and get it merged.
2. Do spot-checks over the next 1-2 weeks to see whether GitHub Actions and CircleCI report the same results. Over the course of this period we should mark the GitHub Actions check "required" and change the status of the CircleCI check to "unrequired".
3. In 1-2 weeks, if everything is working fine, make a separate PR that removes the CircleCI stuff.

Reply:
Sounds good, I've placed it back.